.github/workflows/cicd-production.yaml

```diff
@@ -0,0 +1,70 @@
+name: CI/CD Production
+
+on:
+  deployment
+
+jobs:
+  build-and-test:
+    name: Build and test app
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+      REACT_APP_MAD_LEARNING_API: mad-learning-api
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '12'
+    - run: npm ci
+    - run: npm run build
+    - run: npm test
+  build-and-publish-image:
+    name: Build and publish Docker image
+    needs: build-and-test
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY_NAME: itetechmadacr
+      CLUSTER_NAME: ite-tech-mad-aks
+      CLUSTER_RESOURCE_GROUP: ite-tech-mad-aks-rg
+      NAMESPACE: production
+    steps:
+      - uses: actions/checkout@master
+      # Connect to Azure Container registry (ACR)
+      - uses: azure/docker-login@v1
+        with:
+          login-server: ${{ env.REGISTRY_NAME }}.azurecr.io
+          username: ${{ secrets.REGISTRY_USERNAME }} 
+          password: ${{ secrets.REGISTRY_PASSWORD }}
```

Comment:
Will create credentials and send

Reply:
Actually, you can just set the project secret directly, then I won't need to touch it 😛

Reply:
Right, I named these differently